ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,226 @@
+                                <seg type="script">19th-century script</seg> 
+                                <desc> <persName
+                                    ref="PRS1071dAbbadi">A. d'Abbadie</persName> describes the handwriting as <foreign xml:lang="fr">'fine'</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the front board.
+                                </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the inner side of the front board.
+                                    </desc>
+                                </item>
+                            </list>
+                            </additions>
+                        <bindingDesc>
+                            <binding  contemporary="true">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>  
+                                <decoNote xml:id="b2" type="Cover">Leather quarter cover.</decoNote>  
+                                <decoNote xml:id="b3" type="SewingStations" cert="medium">4</decoNote>  
+                                <decoNote xml:id="b4" cert="medium">Two pairs of sewing stations.</decoNote>  
+                                <decoNote xml:id="b5" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                </decoNote>  
+                                <decoNote xml:id="b6" type="SlipCase">According to <persName
+                                    ref="PRS1071dAbbadi">A. d’Abbadie</persName>,
+                                    the manuscript has a new traditional two-part leather case.</decoNote>              
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was commissioned by  <persName
+                                ref="PRS1071dAbbadi">A.d'Abbadie</persName> in Ethiopia. According to him, the text of the manuscript was copied for him for <foreign xml:lang="fr">'réparer l'étrange oubli qu'on avait fait de cette section dans le manuscrit n. 87'</foreign>.
```

Comment:
space between `A.` and `d'Abbadie`

Reply:
It would probably be more user friendly to translate French and other languages to English and give the quotation in the original language in brackets.